src/main/java/com/blamejared/ambientenvironment/AmbientEnvironment.java

```diff
@@ -16,96 +16,98 @@
 import org.apache.commons.lang3.tuple.Pair;
 
 @Mod("ambientenvironment")
-public class AmbientEnvironment {
+public final class AmbientEnvironment {
```

Comment:
Is this really needed? It just feels like you're adding final for the same of adding final.